shared/package.json

```diff
@@ -126,6 +126,8 @@
     "emoji-mart": "2.9.2",
     "exif-js": "2.3.0",
     "expo-contacts": "5.0.1",
+    "expo-image-picker": "git://github.com/MarcoPolo/expo-image-picker",
```

Comment:
Needed for now because https://github.com/expo/expo/pull/4717 isn't in the published version yet.

Happy to move to a keybase repo (I didn't have permissions for that), but I don't think we'll need this for long.